source/integrate_with_govuk_pay/index.html.md.erb

```diff
@@ -0,0 +1,170 @@
+
+* automatically fetch data about the outcome of payment journeys, and import
+that into your finance system so that payments can be reconciled against bank
+transaction information
+* connect a customer-relationship management (CRM) or case-management system
+to GOV.UK Pay, so that staff can issue refunds from within your system.
+
+You may want to read more in the sections on [__Reporting__](/reporting) and
+[__Refunding payments__](/refunding_payments).
+
+### APIs (GOV.UK Pay)
+
+The public GOV.UK Pay API offers a set of simple operations to conduct and
+report on payments. You can read about the API in the [__API
+reference__](/api_reference) section of this documentation and use the
+interactive [API explorer](https://gds-payments.gelato.io/api-explorer/)
+[external link]. You can also read more about [__Reporting__](/reporting).
+
+## Mop-up jobs
+
+A mop-up job is a background process which checks the outcome of incomplete
+payment journeys.
+
+Most services need to release their service to the user automatically, upon
+successful completion of a payment. If you do this then you should plan for
+inevitable failures associated with unreliable internet connections and user
+behaviour. For example:
+
+* user abandons their payment journey before completing it
+* user completes their payment successfully, but their network connection is
+interrupted before they return to your service
+
+In these cases, users will never visit the `return_url` on your service. You
+should therefore not rely on the `return_url` to determine the outcome of the
+payment journey.
+
+### Using a mop-up job
+
+This section describes the recommended approach for most integrations. It
+provides the best guarantee that edge cases are handled properly and
+automatically.
+
+In order to use a mop-up job, you should have a datastore which keeps track of
+incomplete payment journeys.
+
+The following diagram gives a detailed overview of the sequence:
+
+![](images/keep-track-of-incomplete-payment-journeys.png)
+
+The following diagram shows an example of an incomplete payment journey, and
+how a mop-up job would clean it up:
+
+![](images/incomplete-payment-journeys-mop-up-jobs.png)
+
+Your mop-up job may run frequently, for example every half an hour. It should
+ignore recent payments to avoid interfering with in-flight payments. If it
+finds ‘stale’ incomplete payment journeys then it’s likely that payment
+journey has been abandoned. The mop-up job should call the GOV.UK Pay API to
+determine the status of the payment.
+
+### Integrations without mop-up jobs
+
+You may have a very low-volume service where you have a manual process for
+checking that payments succeeded. For example, using the GOV.UK Pay admin
+tool. You may therefore choose not to use a mop-up job. In this case, you
+should make sure your service staff are trained to check the outcomes of
+payments in the GOV.UK Pay admin tool before releasing the service to the end
+user.
```

Comment:
I have a very slight concern about the use of 'trained' here - since it could make it sound like the process is more complicated than it really is. Could 'you should make sure your service staff check the outcomes of payments..' work instead?

Reply:
We agreed: "In this case, you should make sure your service staff are instructed to check the outcomes of payments in the GOV.UK Pay admin tool before releasing the service to the end user."